azurerm/resource_arm_container_group.go

```diff
@@ -80,6 +80,31 @@ func resourceArmContainerGroup() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"image_registry_credential": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"server": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"username": {
```

Comment:
Could we get a blank line between these properties like the rest of the resource?